tests/BUILD.bazel

```diff
@@ -1,17 +1,41 @@
 load("@bazel_skylib//rules:build_test.bzl", "build_test")
-load("//tools/bazel_integration_test:bazel_integration_test.bzl", "bazel_integration_test")
+load("@rules_bazel_integration_test//bazel_integration_test:defs.bzl", "default_test_runner")
+load(":integration_test.bzl", "rules_python_integration_test", "rules_python_integration_test_suite")
 
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])
 
-bazel_integration_test(
+_LEGACY_FLAGS = [
+    "--noenable_bzlmod",
+]
+
+default_test_runner(
+    name = "legacy_test_runner",
```

Comment:
Instead of legacy test runner, how about "workspace_test_runner"? That better describes what its for

Reply:
Done